04-ruby-mvc-sinatra/active-record-intro/readme.md

```diff
@@ -155,15 +159,15 @@ Then I'm going to run `$ bundle install` in the terminal.
 
 ### Setup Ruby - Hospital (You do - 10m)
 
-- Create an `app.rb` file for this application
-- Create a Gemfile
-- Create a folder for your models
-- Create a file that will contain your AR class definition for doctors
-- Load dependencies into Gemfile and then bundle install
+ 1. Create an `app.rb` file for this application
+ 2. Create a Gemfile
```

Comment:
do students know what a gemfile is / what problem it solves? I know they've seen it, just not sure we've given a proper explanantion of it and Gemfile.lock